examples/platforms/nrf528xx/nrf52840/openthread-core-nrf52840-config.h

```diff
@@ -285,7 +285,34 @@
  *
  */
 #ifndef OPENTHREAD_CONFIG_CSL_SAMPLE_WINDOW
-#define OPENTHREAD_CONFIG_CSL_SAMPLE_WINDOW 30
+#define OPENTHREAD_CONFIG_CSL_SAMPLE_WINDOW 10
+#endif
+
+/**
+ * @def OPENTHREAD_CONFIG_PLATFORM_DELAY_TRANSMIT_SUPPORTED
+ *
+ * The platform radio has the ability to transmit after a specified delay time.
+ *
+ */
+#define OPENTHREAD_CONFIG_PLATFORM_DELAY_TRANSMIT_SUPPORTED 1
+
+/**
+ * @def OPENTHREAD_CONFIG_CSL_USE_PLATFORM_DELAY_TRANSMIT
+ *
+ */
+#ifndef OPENTHREAD_CONFIG_CSL_USE_PLATFORM_DELAY_TRANSMIT
+#define OPENTHREAD_CONFIG_CSL_USE_PLATFORM_DELAY_TRANSMIT (OPENTHREAD_CONFIG_THREAD_VERSION >= OT_THREAD_VERSION_1_2)
+#endif
```

Comment:
Suggest to combine this with `OPENTHREAD_CONFIG_MAC_SOFTWARE_TX_TIMING_ENABLE`. Better to remove this define and use `OPENTHREAD_CONFIG_MAC_SOFTWARE_TX_TIMING_ENABLE` like other radio capabilities, like `OPENTHREAD_CONFIG_MAC_SOFTWARE_RETRANSMIT_ENABLE` and `OPENTHREAD_CONFIG_MAC_SOFTWARE_CSMA_BACKOFF_ENABLE`